### PR TITLE
create : Products page created

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
made product and filter section

### CODE REVIEW

**Review by vishalg8454 — APPROVED** [review]
Looks good to merge.

**Review by Rockyraven — APPROVED** [review]
All goods to me You can merge

**Review by sagargse — APPROVED** [review]

File: categories.html [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+                        </label>
+                    </div>
+                </div>
+            </div>
+        </aside>
+        <!-- main content -->
+        <div class="main-content">
+            <div class="card">
+                <img src="./images/homepage/webdev.png" alt="product">
+                <div class="card-container">
```
Comment (VLeads) [inline categories.html:299]: In image you can use --> loading=lazy <-- . It will help in the optimization of your app.

**Review by kushanksriraj — APPROVED** [review]

File: css/categories.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+/* sidebar */
+aside {
+    flex: 1 0 25%;
+    height: calc(100vh - 5rem);
+    padding: 1.5em;
+    position: sticky;
+    top: 5rem;
+    left: 0;
+    overflow-y: auto;
+    overflow-x: hidden;
```
Comment (VLeads) [inline css/categories.css:19]: you can remove overflow-x, if you have specifically used overflow-y there.

**Review by VLeads — APPROVED** [review]
All are good. I have added few suggestions. You can have a look over there.

**Review by Shubhamkr2610 — APPROVED** [review]
Code looks good and clean to me. You are good to go

**Review by Shailseen — APPROVED** [review]
LGTM. you can proceed for merge.

**Review by VLeads — APPROVED** [review]
ecom-review

In products, last product is in center. In my opinion it would look good if it is left aligned. Apart from that, nice work !!

**Review by Developers-codz — APPROVED** [review]
Looks good to merge

File: css/categories.css (hunk 2 above)
Comment (arjun-computer-geek) [reply css/categories.css:19]: I don't want to overflow the content to the x axis that's why I have given

**Pull request merged**